### DISCUSSION TITLE
Is there a way to control whether paragraph text shows in the story bookmarks?

### DISCUSSION THREAD

**Discussion by samkraw** [TerriaJS/terriajs#6943] [Q&A] [answered]
While creating a story I noticed that the paragraph text is sometimes displayed below the title in the story bookmarks, but not always. I am trying to understand how to control this as it seems random at the moment. 
I am copying formatted text into the paragraph editing window, but even though all the text is formatted the same way (in Word), the behaviour in the bookmark is erratic.

Ideally I would like the paragraph text to not show for any of the bookmarks. Second best option is to have consistency, so that if it shows it shows for all bookmarks.

How can I influence whether the paragraph text shows or not?
![image](https://github.com/TerriaJS/terriajs/assets/25005888/21f69148-9de4-4575-99c9-2601949ea159)

**Comment by nf-s** [comment, marked as answer]
Hi @samkraw 

The bookmark is supposed to the first 100 characters of the paragraph text. It does this by looking at the HTML, finding the first element, and then picking first 100 characters.

Copy and pasting from Microsoft Word could add extra HTML elements - so the first element may not contain your text.

For example

```
<p></p>
<p>Some Text</p>
```

We currently don't have a way to edit HTML directly in the story body. But if you copy and paste your text into a plain-text or code editor, then all HTML will be stripped

**Reply by samkraw** [reply]
Thanks! That led me to figure out that, if the text html includes a newline tag, the bookmark doesn't show any text. Next step for me is to figure out how and why Word sometimes adds it.
